test/net/sourceforge/kolmafia/request/FightRequestTest.java

```diff
@@ -2602,4 +2602,17 @@ public void canTrackSurprisinglySweetStab() {
       }
     }
   }
+
+  @Test
+  public void canTrackMimicEggLay() {
+    var cleanups = new Cleanups(withProperty("_mimicEggsObtained", 0), withFight());
```

Comment:
no, this would explode our tests without really offering any value, IMO

It would be nice to have a test with the failure text that confirms the property isn't updated, but it's not essential.

Reply:
OK